packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js

```diff
@@ -143,6 +143,16 @@ const start = zcf => {
     getPool,
   );
 
+  // compatibility with Treasury which expects to be able to collect fees from
+  // the AMM.
+  const makeCollectFeesInvitation = () => {
+    return zcf.makeInvitation(seat => seat.exit(), 'collect empty fee');
```

Comment:
For full compatibility, should we name the invitation the same thing? 
```suggestion
    return zcf.makeInvitation(seat => seat.exit(), 'collect Fees');
```

Reply:
It's easy enough to do, though I don't see any way that it would help. 

Done